packages/cloud-cognitive/src/components/NonLinearReading/NonLinearReading.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { ChevronUp16 } from '@carbon/icons-react';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg } from '../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--non-linear-reading`;
+const componentName = 'NonLinearReading';
+
+// Default values for props
+const defaults = {
+  theme: 'light',
+};
+
+/**
+ * Use non-linear reading when space is limited to share a
+ * brief, at-a-glance, summary of a concept that may require
+ * more explanation for some users.
+ */
+export let NonLinearReading = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+      children,
+      className,
+      /* other props for NonLinearReading, with default values if needed */
+      definition,
+      theme = defaults.theme,
+      ...rest
+    },
+    ref
+  ) => {
+    const [isOpen, setOpen] = useState(false);
+
+    const handleToggle = () => {
+      setOpen((prevState) => !prevState);
+    };
+
+    const handleKeyDown = (event) => {
+      const { key } = event;
+      if (key === ' ' || key === 'Enter') {
+        handleToggle();
+      }
+    };
+
+    return (
+      <span
+        {...rest}
+        className={cx(blockClass, className)}
+        ref={ref}
+        role="main"
+        {...getDevtoolsProps(componentName)}
+      >
+        {' '}
+        <button
+          type="button"
+          className={cx(`${blockClass}--term-${theme}`, [
+            isOpen
+              ? [`${blockClass}--term-${theme}-open`]
+              : [`${blockClass}--term-${theme}-closed`],
```

Comment:
I've never worked on a project that rigidly enforced BEM because it was never _that_ important, and I've never worked on a component library where it was very important. Ergo, I'm not an expert. I shall be guided by you :).

I expect there will be more BEM comments with future Novice to Pro comments.

Reply:
This is true, which is probably why we (including me) haven’t been consistent about it in the past. I might be the only one who will call out BEM though. 😅  Most of our standards are caught through the linting/CI checks.

That said, if only classnames are an issue, I can hold off BEM comments since it wouldn’t be a very productive to just replace these and probably more prone to error if we were to miss one.